tbx/project_styleguide/templates/patterns/navigation/components/includes/subnav-menu.html

```diff
@@ -0,0 +1,37 @@
+{% load wagtailcore_tags %}
+
+<div class="sub-nav sub-nav--{{ number }}" data-subnav>
+    <ul class="sub-nav__list">
+        <li class="sub-nav__item sub-nav__item--back">
+            <a data-subnav-back href="#" class="sub-nav__link sub-nav__link--back">
+                {% include "patterns/atoms/icons/icon.html" with name="arrow" classname="sub-nav__icon-back" %}
+                Back to main menu
+            </a>
+        </li>
+        <li class="sub-nav__item sub-nav__item--parent">
+            <a class="sub-nav__link sub-nav__link--parent" href="{{ parent.url }}">
+                {{ parent.text }}
+            </a>
+        </li>
+        {% for link in links %}
+            <li class="sub-nav__item">
+                {% with children=link.get_children.live.public.in_menu %}
+                    <a
+                        class="sub-nav__link {% if children and not hide_children %}sub-nav__link--has-children{% else %}sub-nav__link--no-children{% endif %}"
+                        {% if children and not hide_children %}
+                            data-open-subnav aria-haspopup="true" aria-expanded="false"
+                        {% endif %}
+                        href="{{ link.url }}"
```

Comment:
I'm wondering what a screen reader would announce if there were children? Assume it would be 'link' but in that instance when clicking it, it would open the next nav level.

Reply:
It does announce 'link'. I could make this into a button if there are children - but then I'd need to do the same in the primary navigation as well - and at the moment it's quite handy that at desktop, if javascript is disabled, then the links still work. I slightly wish I had completely separated out the markup and classes for desktop and mobile, but it's a bit late now. I think the mobile menu generally needs some screen reader improvements as at the moment it is quite confusing. I will suggest to Anna we add a future ticket for that as it's already taken me a while to do the navigaiton.